umbraco-cloud/release-notes/2023-09-releasenotes.md

```diff
@@ -20,3 +21,12 @@ The new "Shared Secrets" panel lets you define secrets that are critical during
 Besides the new shared secrets, we still have the **Environment Secrets**: For secrets that are specific to individual environments, continue to use the existing "Environment Secrets" panel. These secrets remain locked to the selected environment and won't be accessible by others.
 
 For more detailed information on how to leverage secrets, please visit our [documentation](https://docs.umbraco.com/umbraco-cloud/set-up/project-settings/secrets-management).
+
+## Upgraded "Usage" Page for Custom Plans
+We've specifically updated the "Usage" page for those who have added extra bandwidth or media storage to their project plan. Now, you'll see those custom additions accurately displayed. For example, if you're on a Professional plan and have purchased an additional 3000GB of bandwidth, your "Usage" page will now show 3000GB instead of the default 1000GB.
```

Comment:
```suggestion
We've specifically updated the "Usage" page for those who have added extra bandwidth or media storage to their project plan. Now, you'll see those custom additions accurately displayed. For example, if on a Professional plan, you purchase an additional 3000GB of bandwidth, your "Usage" page will show 3000GB instead of the default 1000GB.
```